src/Cms/Blocks.php

```diff
@@ -115,8 +129,31 @@ public static function parse($input): array
 			try {
 				$input = Json::decode((string)$input);
 			} catch (Throwable) {
-				$parser = new Parsley((string)$input, new BlockSchema());
-				$input  = $parser->blocks();
+				// @deprecated try to import the old YAML format
+				// @todo block.converter remove eventually
+				// @codeCoverageIgnoreStart
+				try {
+					$yaml  = Yaml::decode((string)$input);
+					$first = A::first($yaml);
+
+					// check for valid yaml
+					if (
+						empty($yaml) === true ||
+						(
+							isset($first['_key']) === false &&
+							isset($first['type']) === false
+						)
+					) {
+						throw new Exception('Invalid YAML');
+					} else {
+						$input = $yaml;
+					}
+				} catch (Throwable $e) {
+					// the next 2 lines remain after removing block.covnerter
+					$parser = new Parsley((string)$input, new BlockSchema());
+					$input  = $parser->blocks();
```

Comment:
These lines shouldn't be ignored from coverage:

```suggestion
					// the next 2 lines remain after removing block.converter
					// @codeCoverageIgnoreEnd
					$parser = new Parsley((string)$input, new BlockSchema());
					$input  = $parser->blocks();
					// @codeCoverageIgnoreStart
```